src/common/constants/constants.ts

```diff
@@ -2,24 +2,31 @@ import { Genre, Role } from './enums'
 
 export type PageEnum = Genre | Role
 
-export const roleToText: { [key in Role]: string } = {
-  director_of_photography: 'Director of Photography',
-  director_producer: 'Director & Producer',
-} as const
-
-export const roleToUrl: { [key in Role]: string } = {
-  director_of_photography: 'director-of-photography',
-  director_producer: 'director-producer',
-} as const
-
-export const genreToText: { [key in Genre]: string } = {
-  animation: 'Animation',
-  documentary: 'Documentary',
-  series: 'Series',
-} as const
-
-export const genreToUrl: { [key in Genre]: string } = {
-  animation: 'animation',
-  documentary: 'documentary',
-  series: 'series',
+interface EnumMapping {
+  label: string
+  url: string
+}
+export const enumMappings: { [key in Role]: EnumMapping } & {
+  [key in Genre]: EnumMapping
+} = {
+  director_of_photography: {
+    label: 'Director of Photography',
+    url: '/videos?filter=director-of-photography',
+  },
+  director_producer: {
+    label: 'Director & Producer',
+    url: '/videos?filter=director-producer',
+  },
+  animation: {
+    label: 'Animation',
+    url: '/series/lo-phe-life',
+  },
+  documentary: {
+    label: 'Documentary',
+    url: '/videos?filter=documentary',
+  },
+  series: {
+    label: 'Series',
+    url: '/series',
+  },
```

Comment:
Update handling of enums.
Instead of a series of function to do one thing, this map will allow is to quickly get the exact values we want